Misc/NEWS.d/next/Core and Builtins/2021-04-22-10-46-40.bpo-43908.Co3YhZ.rst

```diff
@@ -0,0 +1,3 @@
+Introduce :const:`Py_TPFLAGS_IMMUTABLETYPE` for immutable type objects, and
+modify :c:func:`PyType_Ready` to set it for all static types. Patch by
```

Comment:
PyType_Ready sets Py_TPFLAGS_IMMUTABLETYPE for all non heap types right?

```suggestion
modify :c:func:`PyType_Ready` to set it for all non heap types. Patch by
```

Reply:
Non-heap types are static: https://docs.python.org/3/c-api/typeobj.html#heap-types